physics/module_sf_noahmplsm.F90

```diff
@@ -842,7 +856,9 @@ subroutine noahmp_sflx (parameters, &
                  emissi ,pah    ,canhs,                           &
 		     shg,shc,shb,evg,evb,ghv,ghb,irg,irc,irb,tr,evc,chleaf,chuc,chv2,chb2 )                                            !out
 
-                 qsfc = q1                         !
+    qsfcveg  = eah*0.622/(sfcprs - 0.378*eah)
```

Comment:
0.622 is already defined as a physical constant in the host model and CCPP: `ratio_of_dry_air_to_water_vapor_gas_constants`. This should be used instead of redefining it here, and, of course, the 0.378 number is 1 - the same constant.